src/Service/FormAnnotationBuilderFactory.php

```diff
@@ -96,7 +96,7 @@ private function injectFactory(
         AnnotationBuilder $annotationBuilder
     ) {
         if ($formElementManager instanceof FormElementManagerV2Polyfill) {
-            $formElementManager->injectFactory($annotationBuilder, $container);
+            $formElementManager->injectFactory($annotationBuilder, $formElementManager);
```

Comment:
```
When we refactored the component for v3, we took this into consideration, and decided that plugin managers should be injected with the parent "context" (usually the application-level service manager), and that the parent context should be passed to plugin factories and initializers. This reduces the amount of boilerplate necessary for the vast majority of these classes, as it addresses the most typical use case.
```
Makes sense, I've dont think that anyone wants create another `Controller` in a `ControllerFactory` 😁.

Thanks for your patience and for those great insights of what was changed, e.g.